.checkstyle/import-control.xml

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE import-control PUBLIC
+        "-//Checkstyle//DTD ImportControl Configuration 1.4//EN"
+        "https://checkstyle.org/dtds/import_control_1_4.dtd">
+
+<!-- We do not have rules for all classes. So we have to first allow all classes import everything
```

Comment:
Just to be sure I have got it right, does it mean that if we need to import a new package in a class, we also need to add that "allow" rule here right?

Reply:
The rules apply currently only to the three packages (and its sub-packages) listed in the description. If you change anything else, you do not need to do anything.

If you change something in the model classes for which these rules apply and add some imports, it depends what imports these are. The rules already allow you to import other model classes, API classes, Java classes (such as Maps, Lists, etc.), and so on. So for most changes, you should not need anything. If you add import to something outside of the allowed packages, you should consider whether it is really correct and if yes, you might need to add the package or class here. In general, the thought behind this is mainly that the Model classes should not use any of the operators (resource operators and assembly operators). 

I tried to improve the description in the XML file to make it a bit more clear.

Reply:
Makes sense thanks